src/Instagram/Transport/JsonTransportFeed.php

```diff
@@ -85,9 +88,16 @@ public function fetchData($userName, $limit)
         $newCache = new Cache();
         $newCache->setUserId($cache->getUserId());
         if ($res->hasHeader('Set-Cookie')) {
+            $saveCookies = [];
+            foreach ($res->getHeaders()['Set-Cookie'] as $cookie) {
+                $setCookie = SetCookie::fromString($cookie);
+                if (in_array($setCookie->getName(), $this->requiresCookieKeys, true)) {
+                    $saveCookies[] = $cookie;
+                }
+            }
             $newCache->setCookie($res->getHeaders()['Set-Cookie']);
```

Comment:
I guess you should remove this line?

Reply:
Thank you for your review.
I fixed the issue commented by you. ↓ I used git rebase in practice. 

```diff
 src/Instagram/Transport/JsonTransportFeed.php | 3 +--                                
 1 file changed, 1 insertion(+), 2 deletions(-)               
                                                                                      
diff --git a/src/Instagram/Transport/JsonTransportFeed.php b/src/Instagram/Transport/J
index bb6401d..293757d 100644                                                         
--- a/src/Instagram/Transport/JsonTransportFeed.php                                   
+++ b/src/Instagram/Transport/JsonTransportFeed.php                                   
@@ -95,9 +95,8 @@ class JsonTransportFeed extends TransportFeed                       
                     $saveCookies[] = $cookie;                             
                 }                                                          
             }                                                                        
-            $newCache->setCookie($res->getHeaders()['Set-Cookie']);                  
+            $newCache->setCookie($saveCookies);                                      
         }                                                                            
-        $newCache->setCookie($saveCookies);                                          
         $this->cacheManager->set($newCache, $userName);                 
                                                                               
         return $data->data->user;                         
```

Reply:
👍